src/components/table/index.md.njk

```diff
@@ -41,6 +41,12 @@ When comparing columns of numbers, align the numbers to the right in table cells
 
 {{ example({group: "components", item: "table", example: "numbers", html: true, nunjucks: true, open: false, size: "m"}) }}
 
+## Custom column widths
+
+You can use provided [width override classes](../../styles/spacing#width-override-classes) or use your own to set the width of table columns.
```

Comment:
Might be good to expand this a little bit...

You can use the [width override classes](#) to set the width of table columns. If these don't meet your needs you can set your own by...(how you do it)

?

Reply:
But @amyhupe to re-write...

Reply:
I agree with this approach, but we'd need to remove the contraction, so:

"You can use the width override classes to set the width of table columns. If these do not meet your needs you can set your own by...(how you do it)"

Reply:
ok, can i have the full text then?

Reply:
Neither Dave or I knows how you set your own widths, so we need someone who does to fill in the explanation. Happy to pair with someone on it.

Reply:
...by providing the classes option to the Nunjucks macro as shown in the example